src/packages/client/package.json

```
@@ -37,15 +37,16 @@
   "bugs": "https://github.com/prisma/prisma/issues",
   "scripts": {
     "build": "node helpers/build.js",
-    "test": "jest",
+    "test": "node helpers/patchJest.js && jest",
     "format": "prettier --write .",
     "lint": "eslint --cache --fix --ext .ts .",
     "lint-ci": "eslint --ext .ts .",
     "generate": "node scripts/postinstall.js",
     "postinstall": "node scripts/postinstall.js",
     "prepare": "cp scripts/backup-index.js index.js && cp scripts/backup-index.d.ts index.d.ts",
     "prepublishOnly": "pnpm run build",
-    "precommit": "lint-staged"
+    "precommit": "lint-staged",
+    "patch-package": "patch-package"
```

We could remove since it's not needed?
```suggestion
    "precommit": "lint-staged"
```